templates/upstart/carbon-cache.conf

```diff
@@ -10,7 +10,7 @@ setgid <%= @group %>
 
 respawn
 
-pre-start exec rm -f '<%= @root_dir %>'/storage/carbon-cache.pid
+pre-start exec rm -f '<%= @root_dir %>'/storage/carbon-cache-a.pid
```

Comment:
Unrelated to this change, why are these quotes correct?

When the ERB is processed they look odd:

```
pre-start exec rm -f '/opt/graphite'/storage/carbon-cache.pid
```

I see that this syntax works:

```
$ ls /tmp/foobar/
hello
$ rm -f '/tmp/foobar'/hello
$ ls /tmp/foobar/ # no files
```

But I don't understand why.